snomed/com.b2international.snowowl.snomed.datastore/src/com/b2international/snowowl/snomed/datastore/request/SnomedConceptSearchRequest.java

```diff
@@ -332,18 +323,10 @@ private Expression addSearchProfile(final Expression searchProfileQuery, final E
 			requestBuilder.filterBySemanticTags(semanticTags);
 		}
 		
-		if (containsKey(OptionKey.USE_FUZZY)) {
-			requestBuilder.withFuzzySearch();
-		}
-		
 		if (containsKey(OptionKey.PARSED_TERM)) {
```

Comment:
`parsedTerm` is another specific TermFilter type, please add it to the implementation.